### PR TITLE
feat:(QTM-307) - Includes skin (dark) variations on theme

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-307

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review
- [ ] TypeScript updated

### A11y review
- [ ] A11y checks
  - [ ] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

**Review by matheusrpp — APPROVED** [review]
## Review guide
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review
- [x] TypeScript updated

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] Firefox
  - [x] Chrome
  - [x] Safari
  - [x] Mobile

File: components/Dropdown/Dropdown.jsx [hunk 1]
```diff
@@ -398,14 +364,14 @@ const Dropdown = ({
...
                     {_buttonLabel}
-                    {isOpen ? (
-                      <ArrowUp theme={theme} />
-                    ) : (
-                      <ArrowDown theme={theme} />
-                    )}
+                    <ArrowIcon
+                      name={isOpen ? 'arrow_drop_up' : 'arrow_drop_down'}
+                      theme={theme}
+                    />
```
Comment (tcelestino) [inline components/Dropdown/Dropdown.jsx:374]: 🆒

**Review by tcelestino — APPROVED** [review]
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review
- [x] TypeScript updated

### A11y review
- [ ] A11y checks
  - [x] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] Firefox
  - [x] Chrome
  - [x] Safari
  - [x] Mobile
- [x] Ux review validation

**Review by matheusrpp — APPROVED** [review]

**Review by tcelestino — APPROVED** [review]
- [x] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review
- [x] TypeScript updated

### A11y review
- [ ] A11y checks
  - [x] accessible via keyboard?
  - [ ] Identifiable through assistive technology (screen-reader chrome extension)?
  - [ ] Semantically correct html elements (or if necessary identified by WAI-ARIA)?
  - [ ] Is there a deficiency in color contrast?

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] Firefox
  - [x] Chrome
  - [x] Safari
  - [x] Mobile
- [x] Ux review validation

**Pull request merged**